meshapp/src/ProfileCardCarousel/Swiper.tsx

```diff
@@ -0,0 +1,219 @@
+import { Grid, IconButton } from "@mui/material";
+import profileCard from "./ProfileCard";
+import ArrowBackIcon from "@mui/icons-material/ArrowBack";
+import ArrowForwardIcon from "@mui/icons-material/ArrowForward";
+
+function Swiper() {
+  const profiles = [
+    {
+      image: "https://media.tenor.com/cOvqBWFm4gIAAAAd/svod-thug-shaker.gif",
+      name: "Joe Mrowgan",
+      job: "podcast host",
+      biography: "MEOW MEOW MEOW MEOW MEOW MEOW MEOW MEOW MEOW MEOW MEEO MEW",
+      interests: ["misinformation", "cats", "meow", "el gato", "kitty"],
+      details: [
+        "cat entertainer in training",
+        "B.S in Milk Drinking",
+        "Looking for a Mentor",
+      ],
+    },
+    {
+      image: "https://media.tenor.com/59a5ljZD-uMAAAAd/cat-shut-up.gif",
+      name: "MEOWSTER",
+      job: "milk guru",
+      biography:
+        "BARK BARK WOOF WOOF RUFF RUFF BARK WOOF RUFF RUFF WOOF GRRR BARK BARK RUFF RUFF GRR WOOF WOOF GRRRR MEOW GRRRR MEOW MEOW MEWWWWW WOOOF WOOF GRRR RUFF MEWO MEWO",
+      interests: ["wire fraud", "cats", "meow", "el gato", "kitty"],
+      details: [
+        "cat scammer in training",
+        "B.S in Milk Laundering",
+        "Looking for a Mark",
+      ],
+    },
+    {
+      image:
+        "https://www.wideopenpets.com/wp-content/uploads/sites/6/2018/02/AdobeStock_109255057.jpg",
+      name: "Mr. Paws",
+      job: "Cat Burgular",
+      biography: "MEOW MEOW MEOW MEOW MEOW MEOW MEOW MEOW MEOW MEOW MEEO MEW",
+      interests: ["grand larceny", "cats", "meow", "el gato", "kitty"],
+      details: [
+        "cat burglar in training",
+        "B.S in Milk Robbing",
+        "Looking for a Mentor",
+      ],
+    },
+    {
+      image:
+        "https://images.livemint.com/img/2021/09/08/1140x641/erik-jan-leusink-IbPxGLgJiMI-unsplash_1631079336162_1631079359671.jpg",
+      name: "Mrowster Pawsident",
+      job: "Pawsident",
+      biography:
+        "BARK BARK WOOF WOOF RUFF RUFF BARK WOOF RUFF RUFF WOOF GRRR BARK BARK RUFF RUFF GRR WOOF WOOF GRRRR MEOW GRRRR MEOW MEOW MEWWWWW WOOOF WOOF GRRR RUFF MEWO MEWO",
+      interests: ["insider trading", "cats", "meow", "el gato", "kitty"],
+      details: [
+        "bureaucat in training",
+        "B.S in Milk Peddling",
+        "Looking for a Mentor",
+      ],
+    },
+    {
+      image:
+        "https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcTyFmpv3VDfx13aPqlY__oyW4Xpmi7VoEYo3cMcejNL9A&usqp=CAU&ec=48665698",
+      name: "Alex Mrownes",
+      job: "propagandist",
+      biography: "MEOW MEOW MEOW MEOW MEOW MEOW MEOW MEOW MEOW MEOW MEEO MEW",
+      interests: ["cats", "meow", "fearmongering", "el gato", "kitty"],
+      details: [
+        "cat propagandist in training",
+        "B.S in Milk Stockpiling",
+        "Looking for a Mentor",
+      ],
+    },
+  ];
+
+  //keeps track of the current card we're on
+  const [position, positionSet] = useState(0);
+
+  //moves to the next card if possible
+  const onRight = () => {
```

Comment:
Please rename this and `onLeft` to `goRight` and `goLeft`. Refer to [this article on semantic method-naming](http://codelegance.com/semantic-method-naming/) for some more insight on the motivation for naming functions this way.

Reply:
done, ty for the article